container_create.sh

```diff
@@ -0,0 +1,12 @@
+# Switch back to dialog for any ad-hoc use of apt-get
```

Comment:
-1 on adding this file. I think ideally we shouldn't even have vscode stuff in the repo, and we should be agnostic of what editor or IDE devs are using in this repo. I understand that it can be convenient  to have a settings file for vscode, and we already do, but adding more vscode scripts to the repo seems excesive. If `ci/setup_env.sh` can't be used, and this code can't be added directly to the vscode config file, maybe it's a good time to create a separate project for vscode stuff for pandas development.

Reply:
Yea I agree with Mark here. I think adding stuff specific to an IDE is favoring a workflow that we should probably be agnostic to. Maybe better served as a dedicated pandas-vscode repo that just uses pandas as a submodule?

Reply:
Hmmm. The `.devcontainer.json` is already an editor-specific file to that line has already been crossed.

How about we create a `/.devcontainer` folder and move both the `.devcontianer.json` and `container_create.sh` files into it? This way we don't have these files pollute the workspace.